tests/Aspire.Hosting.Docker.Tests/DockerComposePublisherTests.cs

```diff
@@ -160,18 +162,24 @@ public async Task DockerComposeAppliesServiceCustomizations()
                 // Set a restart policy
                 composeService.Restart = "always";
 
+                composeService.ContainerName = containerNameParam.AsEnvPlaceHolder(serviceResource);
```

Comment:
I'm not sure about the extension name here. Maybe `AsEnvironmentPlaceholder` (note Env => Environment and capitalization of Placeholder).

Reply:
Mark this API as experimental.

Reply:
Oh its already in a preview package - never mind.

Reply:
Yea